tests/sentry/tasks/test_backfill_seer_grouping_records.py

```diff
@@ -2335,7 +2355,28 @@ def test_backfill_seer_grouping_records_cohort_creation_multiple_batches(
     def test_backfill_for_project_with_only_stacktrace(self, mock_post_bulk_grouping_records):
         project = self.create_project(organization=self.organization)
         data = {
-            "stacktrace": {"frames": [{"function": "foo", "filename": "file.php", "in_app": True}]},
+            **ONLY_STACKTRACE,
```

Comment:
It seems I forgot to use it in my previous PR!!

I'm modifying nothing else in this test even though the diff view does not let you notice this.